client/dashboard/me/notifications-extras/index.tsx

```diff
@@ -1,8 +1,177 @@
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import { SectionHeader } from '../../components/section-header';
+import { Text } from '../../components/text';
+import type { WpcomNotificationSettings } from '@automattic/api-core';
+
+const wpcomOptionKeys = [
+	'marketing',
+	'research',
+	'community',
+	'promotion',
+	'news',
+	'digest',
+	'reports',
+	'news_developer',
+	'scheduled_updates',
+] as const;
+
+type WpcomOptionKey = Extract<
+	keyof WpcomNotificationSettings,
+	( typeof wpcomOptionKeys )[ number ]
+>;
+
+const titles: Record< WpcomOptionKey, string > = {
+	marketing: __( 'Suggestions' ),
+	research: __( 'Research' ),
+	community: __( 'Community' ),
+	promotion: __( 'Promotions' ),
+	news: __( 'Newsletter' ),
+	digest: __( 'Digests' ),
+	reports: __( 'Reports' ),
+	news_developer: __( 'Developer Newsletter' ),
+	scheduled_updates: __( 'Scheduled updates' ),
+};
+
+const descriptions: Record< WpcomOptionKey, string > = {
+	marketing: __( 'Tips for getting the most out of WordPress.com.' ),
+	research: __( 'Opportunities to participate in WordPress.com research and surveys.' ),
+	community: __( 'Information on WordPress.com courses and events (online and in-person).' ),
+	promotion: __( 'Sales and promotions for WordPress.com products and services.' ),
+	news: __( 'WordPress.com news, announcements, and product spotlights.' ),
+	digest: __( 'Popular content from the blogs you follow.' ),
+	reports: __( 'Complimentary reports and updates regarding site performance and traffic.' ),
+	news_developer: __( 'A once-monthly roundup of notable news for WordPress developers.' ),
+	scheduled_updates: __( 'Complimentary reports regarding scheduled plugin updates.' ),
+};
+
+const jetpackOptionKeys = [
+	'jetpack_marketing',
+	'jetpack_research',
+	'jetpack_promotion',
+	'jetpack_news',
+	'jetpack_reports',
+] as const;
+
+type JetpackOptionKey = Extract<
+	keyof WpcomNotificationSettings,
+	( typeof jetpackOptionKeys )[ number ]
+>;
+
+const jetpackTitles: Record< JetpackOptionKey, string > = {
+	jetpack_marketing: __( 'Suggestions' ),
+	jetpack_research: __( 'Research' ),
+	jetpack_promotion: __( 'Promotions' ),
+	jetpack_news: __( 'Newsletter' ),
+	jetpack_reports: __( 'Reports' ),
+};
+
+const jetpackDescriptions: Record< JetpackOptionKey, string > = {
+	jetpack_marketing: __( 'Tips for getting the most out of Jetpack.' ),
+	jetpack_research: __( 'Opportunities to participate in Jetpack research and surveys.' ),
+	jetpack_promotion: __( 'Sales and promotions for Jetpack products and services.' ),
+	jetpack_news: __( 'Jetpack news, announcements, and product spotlights.' ),
+	jetpack_reports: __( 'Jetpack security and performance reports.' ),
+};
+
+export default function NotificationsExtras() {
+	const { data } = useQuery( {
+		...meNotificationsSettingsQuery(),
+		meta: { persist: false },
+	} );
+	const mutation = useMutation( meNotificationsExtrasSettingsMutation() );
```

Comment:
We can add a notification to confirm that the save was successful.
I recently learned how to do it here.
 
(https://github.com/Automattic/wp-calypso/pull/105693/files#diff-187240df604a75e07301065bd0042bf47aba8e71e232062d938942e2b16c63d9R42-R44)